### PR TITLE
Lower ginkgo parallelism and move same image rolling-update test out of flaky

### PR DESCRIPTION
**Pull request by j3ffml**
Tentative fix for #25140.

### CODE REVIEW

File: test/e2e/kubectl.go [hunk 1]
```diff
@@ -1054,7 +1054,7 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 		})
 
 		// Flaky issue: #25140
```
Comment (spxtr) [inline test/e2e/kubectl.go]: Remove this comment.

**Comment by spxtr** [comment]
This seems fine to me. I'd like to make a note of how much it actually affects e2e run time.

**Comment by jlowdermilk** [comment]
Sample size only 2, but looks like added time is ~5 min. Other pr-builder runs complete in 30-40min, the two runs for this change were 40 and 46 min.

**Comment by spxtr** [comment]
Lets get one more data point:

@k8s-bot e2e test this issue: #IGNORE

**Comment by spxtr** [comment]
@kubernetes/sig-testing is +5 minutes to e2e jobs too much?

**Comment by ixdy** [comment]
at one point, parallel e2e tests took about 7-8 minutes, so +5 minutes is a lot.

**Comment by spxtr** [comment]
I'd guess it's closer to 20% time increase for the tests (30/25), which should be anywhere from 1-2 minutes. The run times are too inconsistent on PR Jenkins due to quickly-changing loads.

**Comment by jlowdermilk** [comment]
Another option is to bump size of parallel e2e clusters and up ginkgo parallelism to 40 or so. At this point there are 170 tests in the parallel suite so if each one only takes 5 min on average, that's still 30+ minutes counting startup/teardown time.

**Comment by spxtr** [comment]
Hmm, on second thought I think this is probably fine. The +5 min was a pretty wild estimate, really it should be more like +1 min.

**Comment by jlowdermilk** [comment]
@spxtr, what's the verdict on this PR? It may actually help with some of the other kubectl tests that were recently re-enabled and appear to be flaky, but I'm ok with dropping and bumping node count to 4 instead.

**Comment by spxtr** [comment]
I'm fine with it. I'd like @ixdy to give the okay as well, since it will add another couple minutes.

**Comment by ixdy** [comment]
@k8s-bot e2e test this please, issue #IGNORE

**Comment by spxtr** [comment]
9 min 24 sec. Not too bad.

**Comment by spxtr** [comment]
> Tests are more than 96 hours old. Re-running tests.

I don't believe you.

**Pull request merged**